lib/multitask.py

```diff
@@ -279,7 +280,24 @@ def rename(inputs, name):
   num_datasets = len(datasets)
 
   inputs = [tkf.Input(name="%s_in" % x, shape=input_shape) for x in ds_names]
-  mix_weights = [get_mixture_weights(name, inputs[0]) for name in ds_names]
+
+  # splitting MWs to private and shared
+  if share_mw_after is not None and share_mw_after < num_layers:
+    mix_weights = []
+    shared_mws = get_mixture_weights("shared", inputs[0],
+				     start_idx=share_mw_after,
```

Comment:
Wrong indentation. Please check everywhere.

Reply:
Oops, sorry! Pylint missed it for some reason :(